packages/Form/core/src/FieldForm.js

```diff
@@ -140,14 +145,6 @@ class FieldForm extends Component {
     };
   }
 
-  captureMemory() {
```

Comment:
i dont understand why this function have been created, do you know if this function is called somewhere else ?

Reply:
It's not called anywhere else. I think it was created to avoid duplicated code. But with this refacto we have 1 call in `getDerivedStateFromProps` which is static so we cannot use it and 1 in `onFocus` so I removed it for simplicity.

Reply:
ok, you're right to remove it, thank you Xavier